algorithm_sort.rb

```diff
@@ -132,3 +132,30 @@ def sort(words)
 end
 
 sort(words)
+
+# sorts words in the array based on their length
+
+def sort_length(words)
```

Comment:
Using the code line numbers on this pull request as a guide, the spot I'm suggesting to integrate your method is `algorithm_sort.rb::line: 97`.  I know code line numbers can render different in your local IDE, so please let me know if you have any questions!